Source/RunActivity/Viewer3D/WebServices/Web/HUD/index.js

```diff
@@ -86,8 +86,9 @@ function ApiHeadUpDisplay() {
 						DebugWidthTofix = ['primitives','Camera'],
 					// Force information
 						ForceColSpanTo15 = ['Wind Speed'],
-						ForceColSpanTo4 = ['(Simple adhesion model)','Dynamic brake'],
-						ForceColSpanTo3 = ['Axle out force','Loco Adhesion','Wagon Adhesion'],
+						ForceColSpanTo4 = ['(Advanced adhesion model)', '(Simple adhesion model)'],
+						ForceColSpanTo3 = ['Wheel slip (Thres)', 'Conditions', 'Axle drive force', 'Axle brake force', 'Number of substeps', 'Wheel Adh. (Max)',
+							'Axle out force', 'Comp Axle out force', 'Wheel speed (Slip)', 'Wheel ang. pos.', 'Loco Adhesion', 'Wagon Adhesion'],
 					// Brake information
```

Comment:
- If simple adhesion is present, advanced adhesion should also be there.
- "Dynamic brake" is not present in the forces HUD. It is present in the Brakes HUD (part of a pre-formated string) and in the common part (uses different presenation code).
- The existing colspan-3 fields may all have multiple values (per axle), although that does not seem to ever happen. The fields I added also have per-axle values.

Reply:
Agreed